src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -365,11 +365,11 @@ typedef struct glb_lb_policy {
   /** are we already watching the LB channel's connectivity? */
   bool watching_lb_channel;
 
-  /** is \a lb_call_retry_timer active? */
-  bool retry_timer_active;
+  /** is \a lb_call_retry_timer active and its callback pending? */
```

Comment:
It's really not about the timer being active, only about the callback being pending.

Same for all of these fields.

Reply:
I guess the original author meant "the timer is counting down"?

Anyways, I agree that it's sufficient to say the callback is pending.